src/core/router/router.component.tsx

```diff
@@ -18,27 +38,27 @@ export const RouterComponent: React.FunctionComponent = () => {
         <Route
           exact={true}
           path={switchRoutes.configureVerbs}
-          component={ConfigureVerbsScene}
+          render={location => handleRedirect(ConfigureVerbsScene, location)}
         />
         <Route
           exact={true}
           path={switchRoutes.finalScore}
-          component={FinalScoreScene}
+          render={location => handleRedirect(FinalScoreScene, location, 'fromTest')}
```

Comment:
Mmm... I think we can just use an interceptor and avoid having to code the redirecto on each route.

Some tips (in this case this is applied to Auth, but we can use a similar approach):

https://lemoncode.net/lemoncode-blog/2020/12/20/react-router-y-rutas-autorizadas-i

Then, a possible simple way to check if we need to return home, we could simply check, _verbsContext_ if the verbsContext array is null, or empty then jump back to home page.

What do you think?

Reply:
Please check PR #77. I though that is better to separate both proposals till we choose one of them.

Reply:
Finally we will for for PR #77, thanks a lot Vicente and Joaquin it has been great working on this and coming into a final clean solution, we will go through it on next Thursday session